src/barChart.ts

```diff
@@ -1,740 +1,728 @@
-        header.setAttribute("class", "LandingPage");
-        let p1 = document.createElement("a");
-        p1.setAttribute("class", "LandingPageHelpLink");
-        p1.textContent = "Learn more about Landing page";
-
-        p1.addEventListener("click", () => {
-            this.host.launchUrl("https://microsoft.github.io/PowerBI-visuals/docs/overview/");
-        });
-
-        div.appendChild(header);
-        div.appendChild(p1);
-
-        return div;
-    }
-
-    private getColorValue(color: Fill | string): string {
-        // Override color settings if in high contrast mode
-        if (this.host.colorPalette.isHighContrast) {
-            return this.host.colorPalette.foreground.value;
-        }
-
-        // If plain string, just return it
-        if (typeof (color) === 'string') {
-            return color;
-        }
-        // Otherwise, extract string representation from Fill type object
-        return color.solid.color;
-    }
-
-    private initAverageLine() {
-        this.averageLine = this.svg
-            .append('g')
-            .classed('averageLine', true);
-
-        this.averageLine.append('line')
-            .attr('id', 'averageLine');
-
-        this.averageLine.append('text')
-            .attr('id', 'averageLineLabel');
-    }
-
-    private handleAverageLineUpdate(height: number, width: number, yScale: ScaleLinear<number, number>) {
-        let average = this.calculateAverage();
-        let fontSize = Math.min(height, width) * BarChart.Config.xAxisFontMultiplier;
-        let chosenColor = this.getColorValue(this.barChartSettings.averageLine.fill);
-        // If there's no room to place lable above line, place it below
-        let labelYOffset = fontSize * ((yScale(average) > fontSize * 1.5) ? -0.5 : 1.5);
-
-        this.averageLine
-            .style("font-size", fontSize)
-            .style("display", (this.barChartSettings.averageLine.show) ? "initial" : "none")
-            .attr("transform", "translate(0, " + Math.round(yScale(average)) + ")");
-
-        this.averageLine.select("#averageLine")
-            .style("stroke", chosenColor)
-            .style("stroke-width", "3px")
-            .style("stroke-dasharray", "6,6")
-            .attr("x1", 0)
-            .attr("x1", "" + width);
-
-        this.averageLine.select("#averageLineLabel")
-            .text("Average: " + average.toFixed(2))
-            .attr("transform", "translate(0, " + labelYOffset + ")")
-            .style("fill", this.barChartSettings.averageLine.showDataLabel ? chosenColor : "none");
-    }
-
-    private calculateAverage(): number {
-        if (this.barDataPoints.length === 0) {
-            return 0;
-        }
-
-        let total = 0;
-
-        this.barDataPoints.forEach((value: BarChartDataPoint) => {
-            total += <number>value.value;
-        });
-
-        return total / this.barDataPoints.length;
-    }
-}
+import "./../style/visual.less";

```

Comment:
Add signing for commit https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/managing-commit-signature-verification

Reply:
Create a clean commit history

Reply:
Update version in:

- pbiviz.json
- package.json
- package-lock.json

Reply:
Update changelog